ebms-send-in/src/main/kotlin/no/nav/emottak/ebms/FagmeldingRoutes.kt

```diff
@@ -0,0 +1,56 @@
+package no.nav.emottak.ebms
+
+import arrow.core.Either
+import arrow.core.raise.either
+import io.ktor.http.HttpStatusCode
+import io.ktor.server.auth.authenticate
+import io.ktor.server.request.receive
+import io.ktor.server.response.respond
+import io.ktor.server.routing.Route
+import io.ktor.server.routing.post
+import io.micrometer.prometheus.PrometheusMeterRegistry
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+import no.nav.emottak.auth.AZURE_AD_AUTH
+import no.nav.emottak.melding.model.SendInRequest
+import no.nav.emottak.melding.model.SendInResponse
+import no.nav.emottak.util.marker
+
+fun Route.fagmeldingRoutes(prometheusMeterRegistry: PrometheusMeterRegistry) {
+    authenticate(AZURE_AD_AUTH) {
+        post("/fagmelding/synkron") {
+            val sendInRequest = try {
```

Comment:
Kunne forsåvidt pakket dette inn i Either / catch også hvis jeg skal flisespikke litt :) Da blir det mer enhetlig

Reply:
Tar det i neste PR så denne ikke henger og rotner til neste uke :+1:

Reply:
Løses her: https://github.com/navikt/send-in-send-out/pull/49